src/view/event-edit.js

```diff
@@ -84,7 +84,13 @@ class EventEditView {
     if (!this._element) {
       this._element  = createElement(this.getTemplate());
 
-      renderNestedElement(this._element , new EventDetailsView(this._event).getElement(), RenderPosition.BEFOREEND);
+      renderNestedElement(this._element, new EventDetailsView(this._event).getElement(), RenderPosition.BEFOREEND);
+
+      Array.from(this._element.querySelectorAll('.event__type-input')).forEach((eventTypeInput) => eventTypeInput.addEventListener('click', (evt) => {
```

Comment:
```suggestion
      this._element.querySelectorAll('.event__type-input').forEach((eventTypeInput) => eventTypeInput.addEventListener('click', (evt) => {
```
`forEach` доступен в коллекции нод)

Reply:
Я почему-то думала недоступен(( Исправила